### PR TITLE
Commit for graph colours

### PR DESCRIPTION
**Pull request by Miyuki0**
Changes the colours of the graph both in homepage (index.html) and in meta-analysis page.

### CODE REVIEW

File: webpages/js/metaanalyses.js [hunk 1]
```diff
@@ -946,7 +946,13 @@
 
     var groupT = _.findEl(plotEl, 'template.group-grapes');
     var groupTooltipsT= _.findEl(plotEl, 'template.group-tooltips');
+
+    var currentGroup = -1;
+
     groups.forEach(function (group, index) {
+
+      currentGroup = currentGroup + 1;
```
Comment (jacekkopecky) [inline webpages/js/metaanalyses.js]: please use `+= 1` here, it's more readable

also, please consider starting from 0 on line 950 and putting this increment after the value is used on line 998

File: webpages/js/metaanalyses.js [hunk 2]
```diff
@@ -989,6 +995,8 @@
         grapeEl.setAttribute('r', getGrapeRadius(exp.wt));
         _.setAttrs(tooltipEl, '.grape', 'r', getGrapeRadius(exp.wt));
 
+        grapeEl.classList.add('c' + currentGroup%7);
```
Comment (jacekkopecky) [inline webpages/js/metaanalyses.js]: the number 7 needs to come from a dataset value please, so we have the number of groups and their styles in one place in the SVG template

File: webpages/profile/metaanalysis.html [hunk 3]
```diff
@@ -756,6 +756,30 @@ <h1 class="title editing oneline" contenteditable placeholder="enter a short nam
...
+        fill:#7889AC;
+      }
+      .grape.c5{
+        fill:#847FB3;
+      }
+      .grape.c6{
+        fill:#9975AD;
+      }
+
+
```
Comment (jacekkopecky) [inline webpages/profile/metaanalysis.html]: too many empty lines, one will do

File: webpages/index.html [hunk 4]
```diff
@@ -184,6 +184,14 @@ <h1 class="logo">LiMA</h1> <h2>Living Meta-Analysis</h2>
           fill: #8ddce5;
         }
 
+        .grape.c0 {
```
Comment (jacekkopecky) [inline webpages/index.html:187]: please fix the indentation of the following few lines

**Review by jacekkopecky — CHANGES_REQUESTED** [review]
thanks, the requested changes are mostly just code style comments

File: webpages/js/metaanalyses.js (hunk 1 above)
Comment (Miyuki0) [reply webpages/js/metaanalyses.js]: Yep, the '-1' thing is just an old habit !

File: webpages/index.html (hunk 4 above)
Comment (rragoomundun) [reply webpages/index.html:187]: Done

File: webpages/profile/metaanalysis.html (hunk 3 above)
Comment (rragoomundun) [reply webpages/profile/metaanalysis.html]: Done

File: webpages/js/metaanalyses.js (hunk 2 above)
Comment (rragoomundun) [reply webpages/js/metaanalyses.js]: Done

File: webpages/js/metaanalyses.js (hunk 1 above)
Comment (rragoomundun) [reply webpages/js/metaanalyses.js]: Done

**Comment by jacekkopecky** [comment]
Thanks!

Merged in e7a678e00373bde32e2d636e11a6a9a880ead980

**Pull request closed without merging**